app/editor/src/components/tool-bar/ToolBarSection.tsx

```diff
@@ -0,0 +1,27 @@
+import { Row } from 'tno-core';
+
+import * as styled from './styled';
+
+export interface IToolBarSectionProps {
+  actions: React.ReactNode;
```

Comment:
If I'm not mistaken, this should follow common standards and be named `children`